packages/api/src/command/medical/patient/convert-fhir-bundle.ts

```diff
@@ -44,17 +44,20 @@ export async function handleBundleToMedicalRecord({
   dateTo?: string;
   conversionType: ConsolidationConversionType;
 }): Promise<Bundle<Resource>> {
-  const isSandbox = Config.isSandbox();
-
-  if (isSandbox) {
+  if (Config.isSandbox()) {
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    const bucketName = Config.getSandboxSeedBucketName()!;
     const patientMatch = getSandboxSeedData(patient.data.firstName);
-    const url = await processSandboxSeed({
-      firstName: patientMatch ? patient.data.firstName : "jane",
-      conversionType,
-      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-      bucketName: Config.getSandboxSeedBucketName()!,
+    const patientNameLowerCase = patientMatch ? patient.data.firstName.toLowerCase() : "jane";
+    const fileName =
+      conversionType === "pdf"
+        ? `${patientNameLowerCase}_MR.html.pdf`
+        : `${patientNameLowerCase}_MR.html`;
```

Comment:
Can't we call `createMRSummaryFileName`? It's better to have these all concentrated in one place as much as possible.

Reply:
these sandbox MR summaries are following a different naming pattern, i.e. `jane_MR.json`

Reply:
But can use the `createSandboxMRSummaryFileName`